documentation.md

```diff
@@ -39,25 +39,26 @@ You may find a quick overview about the [main features](/features.html) useful.
 ### API Documentation
 We provide several APIs to interact with SirixDB.
 
-1. The [transactional cursor API](/transactional-cursor-api.html) is a powerful low-level API.
```

Comment:
It looks like the only change here is the removal the the link. Why remove the link? Please change this back.